docs/development/styles.md

```diff
@@ -0,0 +1,58 @@
+# Styles
+
+## Recommended Components
+
+**TODO**
+
+## Guidance
+
+**TODO**
```

Comment:
mention:
need for out of box style system
or, make your own (if you have design team that may be working on a style system)

Reply:
mention:
goal of the styles, who or what target device demographic is. mobile, desktop, browser types, browser versions,etc